pkg/filter/dt/filter_mysql.go

```diff
@@ -95,49 +95,62 @@ func (f *_mysqlFilter) GetName() string {
 }
 
 func (f *_mysqlFilter) PreHandle(ctx context.Context, conn proto.Connection) error {
+	var err error
 	bc := conn.(*driver.BackendConnection)
 	commandType := proto.CommandType(ctx)
 	if commandType == constant.ComStmtExecute {
 		stmt := proto.PrepareStmt(ctx)
 		if stmt == nil {
+			mysqlFilterErrorCount.WithLabelValues("unknown", "pre").Inc()
```

Comment:
metrics should create a new filter, `/pkg/filter/dt/filter_mysql` only process distributed transaction problem.

Reply:
label can be db name、table name、sql type(select、update、delete、insert)